### PR TITLE
Updated docs to cater for Azure Storage Persistences v7 connection string changes

### PR DESCRIPTION
**Pull request by WilliamBZA**
Connects to https://github.com/Particular/NServiceBus.AzureStoragePersistence/pull/34
Connects to https://github.com/Particular/NServiceBus.AzureStoragePersistence/pull/33

Updates the docs and the snippets to highlight the changes made in those PRs.

### CODE REVIEW

**Comment by WilliamBZA** [comment]
Ping @Particular/docs-maintainers

File: nservicebus/azure/azure-storage-persistence.md [hunk 1]
```diff
@@ -45,18 +45,18 @@ snippet:AzurePersistenceFromAppConfig
 
 The following settings are available for changing the behavior of saga persistence through the `AzureSagaPersisterConfig`section:
 
-- `ConnectionString`: Allows you to set the connectionstring to the storage account for storing saga information, defaults to `UseDevelopmentStorage=true`
+- `ConnectionString`: Allows you to set the connectionstring to the storage account for storing saga information, defaults to `UseDevelopmentStorage=true` in Versions 6 and less, and defaults to `null` in Versions 7 and up.
```
Comment (indualagarsamy) [inline nservicebus/azure/azure-storage-persistence.md]: @WilliamBZA - See the Versions language here:
https://github.com/particular/docs.particular.net#version-language
Its Versions 6 and above. Can you make this consistent in other places as well?

Comment (indualagarsamy) [inline nservicebus/azure/azure-storage-persistence.md]: Also avoid language with: you, we, etc. See: https://github.com/particular/docs.particular.net#language-preferences

Comment (WilliamBZA) [inline nservicebus/azure/azure-storage-persistence.md]: Thanks for the link. The personal tone was already there so I left it.

I'll change to reflect.

Comment (indualagarsamy) [inline nservicebus/azure/azure-storage-persistence.md]: @WilliamBZA - awesome. thanks.

Comment (WilliamBZA) [inline nservicebus/azure/azure-storage-persistence.md]: @indualagarsamy Updated :smile:

File: Snippets/Snippets_6/Azure/Persistence/AzureStorage/Usage.cs [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            endpointConfiguration.UsePersistence<AzureStoragePersistence>();
+
+            #endregion
+        }
+
+        #region PersistenceWithAzureHost 7
+
+        public class EndpointConfig : IConfigureThisEndpoint
+        {
+            public void Customize(EndpointConfiguration configuration)
```
Comment (SimonCropp) [inline Snippets/Snippets_6/Azure/Persistence/AzureStorage/Usage.cs]: can we name the param endpointConfiguration  to keep it consistent with all the other doco

File: Snippets/Snippets_6/Snippets_6.csproj [hunk 3]
```diff
@@ -135,6 +139,10 @@
       <HintPath>..\..\packages\NServiceBus.AzureBlobStorageDataBus.0.2.0-unstable0016\lib\net452\NServiceBus.AzureBlobStorageDataBus.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="NServiceBus.AzureStoragePersistence, Version=0.0.0.0, Culture=neutral, PublicKeyToken=9fc386479f8a226c, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\NServiceBus.AzureStoragePersistence.7.0.0\lib\net452\NServiceBus.AzureStoragePersistence.dll</HintPath>
```
Comment (SimonCropp) [inline Snippets/Snippets_6/Snippets_6.csproj]: shouldnt this be an unstable nuget?

File: nservicebus/azure/azure-storage-persistence.md [hunk 4]
```diff
@@ -45,44 +45,48 @@ snippet:AzurePersistenceFromAppConfig
...
 - It only allows 2 simultaneous connections to a single IP address by default
 - It's algorithm stack has been optimized for larger payload exchanges, not for small requests
 - It doesn't trust the remote servers by default, so it verifies for revoked certificates on every request
 
-You can drastically improve performance by overriding these settings. You can leverage the ServicePointManager class for this end and change it's settings, but this must be done before your application makes any outbound connection, so ideally it's done very early in your application's startup routine.
+Performance can be drastically improves by overriding these settings. The ServicePointManager class can be used for this end and change it's settings, but this must be done before the application makes any outbound connection, so ideally it's done very early in the application's startup routine.
```
Comment (SimonCropp) [inline nservicebus/azure/azure-storage-persistence.md]: improved

File: Snippets/Snippets_6/Snippets_6.csproj (hunk 3 above)
Comment (WilliamBZA) [inline Snippets/Snippets_6/Snippets_6.csproj]: Yes, bad monkey patch on my side.

**Comment by WilliamBZA** [comment]
@SimonCropp updated with your changes

**Comment by WilliamBZA** [comment]
Ping @Particular/docs-maintainers

**Pull request merged**